### DISCUSSION TITLE
Your client application uses libraries for user authentication or authorization that will soon be deprecated. See the [Migration Guide](https://developers.google.com/identity/gsi/web/guides/gis-migration) for more information.

### DISCUSSION THREAD

**Discussion by Gopal6696** [abacritt/angularx-social-login#557] [Q&A]
Your client application uses libraries for user authentication or authorization that will soon be deprecated. See the [Migration Guide](https://developers.google.com/identity/gsi/web/guides/gis-migration) for more information.

"angularx-social-login": "^4.0.1",
"@angular/common": "^13.3.3",
"@angular/compiler": "^13.3.3",
"@angular/core": "^13.3.3",

**Comment by Heatmanofurioso** [comment]
@Gopal6696 What version of the library are you using?

**Reply by samYnsat** [reply]
I have the same problem. I am using "angularx-social-login": "^4.1.0"

**Reply by Heatmanofurioso** [reply]
The latest version isn't that.
It should be `"@abacritt/angularx-social-login": "^1.2.4"`
